tests/attack/test_mod_wapp.py

```diff
@@ -87,16 +87,18 @@ async def test_url_detection():
             (
                 'additional',
                 (
-                    '{"name": "Microsoft ASP.NET", "versions": [], "categories": ["Web '
+                    '{"name": "Microsoft ASP.NET", "versions": [], "cpe": '
+                    '"cpe:2.3:a:microsoft:asp.net:*:*:*:*:*:*:*:*", "categories": ["Web '
```

Comment:
What about doing a `rstrip(":*")` on the CPEs extracted from the wappalizer database to get a shortened version ?

Reply:
It's a norm and I don't think it's worth deviating from it.